### PR TITLE
Various updates

### PR DESCRIPTION
**Pull request by geoffwhittington**

### CODE REVIEW

File: install/setup.iss [hunk 1]
```diff
@@ -0,0 +1,122 @@
+#define EXE_VERSION "1.0"
```
Comment (funkaoshi) [inline install/setup.iss]: I don't really understand any of this. FYI.

File: sdetools/sdelib/conf_mgr.py [hunk 2]
```diff
@@ -169,7 +169,7 @@ def __getitem__(self, key):
 
     def __setitem__(self, key, val):
         if key not in self.settings:
-            raise KeyError, 'Unknown configuration item: %s' % (key)
+            return
```
Comment (funkaoshi) [inline sdetools/sdelib/conf_mgr.py]: Any reason we drop this? Might be nice to log a warning for bad configuration items. Would catch typos and the like.

File: setup.py [hunk 3]
```diff
@@ -22,7 +22,11 @@ def set_py2exe_options():
 
     for mod_name in modules.__all__:
         ext_opt['options']['py2exe']['includes'].append('sdetools.modules.%s' % mod_name)
-    
+
+    from sdetools import extlib
```
Comment (funkaoshi) [inline setup.py]: Is there a circular import if this is outside the function?

File: install/setup.iss (hunk 1 above)
Comment (geoffwhittington) [inline install/setup.iss]: That's fine. This is a functioning installer but its not perfect.

This particular macro is for updating the file version of the generated setup.exe. I included it just to be able to differentiate

Comment (funkaoshi) [inline install/setup.iss]: Cool. Yeah, I mean I think I understand what it's doing, but I don't think i can offer you any good commentary on whether it's amazing or not. It probably is.

**Comment by funkaoshi** [comment]
Seems fine @geoffwhittington

File: setup.py [hunk 4]
```diff
@@ -22,7 +22,11 @@ def set_py2exe_options():
```
Comment (funkaoshi) [inline setup.py]: Explain what this does maybe? I don't think i'd know off hand.

**Pull request merged**